public/views/partials/footer.html

```diff
@@ -0,0 +1,17 @@
+<footer id="footer">
+	<div class="scroll-to-top">
+		<a href="" onclick="scrollToTop();">Top</a>
+	</div>
+	<div class="footer-container">
+		<ul class="footer-link-collection">
+            <li><a class="footer-link" href="/contact">Contact Us</a></li>
+            <li><a class="footer-link" href="/media">Social Media</a></li>
+            <li><a class="footer-link" href="/sponsors">Sponsors</a></li>
+            <li><a class="footer-link" href="/links">Links</a></li>
+        </ul>
+        <hr id="footer-bar">
```

Comment:
the <hr> tag id is being referenced in the css, but I'll change these to classes